### DISCUSSION TITLE
What should I do if I just want to move task bar to right of the screen?

### DISCUSSION THREAD

**Discussion by 117649** [valinet/ExplorerPatcher#157] [Q&A]
While everything else is as it is of default WIN11?

**Comment by valinet** [comment]
It's a built-in feature of Windows: right click the taskbar, uncheck "Lock all taskbars", drag it to whichever side you want, optionally right click and check back "Lock all taskbars".

**Reply by 117649** [reply]
I think we are talking about WIN11.

**Reply by valinet** [reply]
Can you be more verbose? Let's stop wasting each other's time. You want to do that with Windows 11's taskbar? I don't know how to do it. This is a forum for questions regarding ExplorerPatcher, which enables the Windows 10 taskbar, not random stuff.

**Reply by llandyw** [reply]
A permanent move requires an edit of the registry.

At key: HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\StuckRects3
Double click the settings key.
In the second row, there's a value of 03 in the fifth place.  This can be changed.
00 left side of screen
01 is top of screen
02 is right side of screen
03 is bottom of screen

There have been reports of losing icons if moved to left or right.  But you can always replace the value with 03 to go back to the bottom.

To change the value, highlight both characters and then type the new value.  If you don't do this, you'll have to move the cursor to the right of the 03, and delete it, otherwise the values will be offset by 1 place, and no telling what that will do.

This works even when MS changes stuff and breaks EP.

Andy
